npm-debug.log

```diff
@@ -0,0 +1,201 @@
+0 info it worked if it ends with ok
```

Comment:
Not quite clear why this is included -- if it's safe to remove, can you strip it?